portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,42 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-/**
- * Adds a random greeting to the page.
+/** 
+ * FUNCTIONS 
```

Comment:
We know these are function, so this comment is not meaningful and we should have better comments describing what these methods do (and each comment should be above each method).

Also, if you do want to group things together, e.g, those lines that setup onclick listeners, you can have a function called `setupOnClickListener` and put those lines into the function, such that you can have a comment for that one method, and it will also make the code in this file more structural

Reply:
Oh I see... Thanks for the suggestion! (: However, won't this make the function extremely long?

Reply:
Function being long should not be too much of concern, especially that I don't see it being super long right now. Also, if it becomes long, then it's time to separate things again, e.g, you can have one function for each setup of onclick.

The main thing is that right now without functions, everything is global variable, which is not necessary and would be more error-prone